File: public/phase-2-20160620022203/C/inmates.c
// Name : Inmates
#include <stdio.h>
int main() {
	// input:
	// n : number of prisoners
	// t : crime threshold
	// c : selected inmates
	// severity :  severity of the crime committed by each inmate
	int n, t, c, severity[10000], result=2, index;
	scanf("%d %d %d", &n, &t, &c);
	for(index = 0; index< n; index++) {
		scanf("%d", &severity[index]);
	}


	// write your code here
	// store your results in `result`

	// output
	printf("%d", result);
	return 0;
}

File: public/phase-2-20160620022203/C_SHARP/inmates.cs
// Name : Inmates
using System;
namespace Application {
	class inmates {
		public static void Main(string[] args) {
		// input:
		// n : number of prisoners
		// t : crime threshold
		// c : selected inmates
		// severity :  severity of the crime committed by each inmate
		int index;
		string[] elements = (Console.ReadLine()).Split(' ');
		int n = Convert.ToInt32(elements[0]);
		int t = Convert.ToInt32(elements[1]);
		int c = Convert.ToInt32(elements[2]);
		int[] severity = new int[n];
		string[] severity_elements = (Console.ReadLine()).Split(' ');
		for(index=0;index<n;index++)
			severity[index] = Convert.ToInt32(severity_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=2;

		Console.WriteLine(result);

		}	
	}
}